internal/kubeconfig/handler.go

```diff
@@ -75,7 +76,7 @@ func (h *Handler) GetKubeconfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if broker.IsOwnClusterPlan(instance.ServicePlanID) {
+	if h.ownClusterPlanID == instance.ServicePlanID {
```

Comment:
Why is this line introduced?

Reply:
If we leave broker.IsOwnClusterPlan we create an import cycle